pkg/systemd/quadlet/quadlet.go

```diff
@@ -466,6 +468,9 @@ func ConvertContainer(container *parser.UnitFile, names map[string]string, isUse
 	// force it, -i makes it ignore non-existing files.
 	serviceStopCmd := createBasePodmanCommand(container, ContainerGroup)
 	serviceStopCmd.add("rm", "-v", "-f", "-i", "--cidfile=%t/%N.cid")
+	if stopTimeout, ok := container.Lookup(ContainerGroup, KeyStopTimeout); ok && len(stopTimeout) > 0 {
+		serviceStopCmd.addf("--time=%s", stopTimeout)
+	}
```

Comment:
Do we even have to set this param on rm? `--stop-timeout` is a option on podman create/run as well and unless `-t` is given on stop/rm it should default to this value.

Reply:
Good point. I was not aware of `--stop-timeout` and just assumed I should use a flag in `rm`. Since all flags (aside from the global one) are used for the `run` command, I'll make this one the same.

Reply:
Done